### DISCUSSION TITLE
Confusion between MyST-Parser and MyST-md

### DISCUSSION THREAD

**Discussion by Mathnerd314** [executablebooks/meta#1194] [General]
I was very confused, the websites https://mystmd.org/ and https://myst-parser.readthedocs.io/ look very similar. I was wondering if it was maybe someone had made an unauthorized copy of the website but it seems it is more complex - myst-parser is the Python version and mystmd is a newer (beta?) JavaScript version. It is not obvious though because both editions mainly refer to themselves as MyST, and they have different features - e.g. I was looking at bibliographies, and mystmd has a nice syntax which is not implemented in myst-parser (https://github.com/executablebooks/MyST-Parser/issues/511).

I think they should be renamed again, myst-parser -> MyST-Py and mystmd -> MyST-JS. This makes it clear that they are two implementations of "the same thing" but may differ somewhat in details. And then it would be better if there was one main site documenting all of the features of MyST but with notes on caveats regarding Py vs. JS. I guess also specific websites individually for MyST-Py and MyST-JS discussing the API and usage, but stuff on authoring should not be duplicated with the main site.

[mystmd.org](https://mystmd.org/) already is an implementation-neutral domain name, so concrete suggestions:

* https://mystmd.org/ - adjust to prominently mention multiple implementations. Currently it only hints at it with the "MyST Ecosystem" section, and the images are not clickable to the different sub-projects.
* https://mystmd.org/guide - put MyST-Py and MyST-JS on equal footing, do not assume that mystmd-js is the only implementation. Remove note "the following documentation refers only to the Javascript MyST Markdown CLI" and mark all JS-specific info individually.
* https://mystmd.org/guide/quickstart, other Quickstart pages - s/MyST/MyST-JS/, move under https://mystmd.org/myst-js/
* https://mystmd.org/guide/typography and other 'Authoring' pages - seem generally fine, but should be clearer on what are Sphinx (MyST-Py) and MyST-JS features 
* Executable Content, Websites, Documents, Extensions - these seem MyST-JS specific and should be under https://mystmd.org/myst-js/
* Reference - a mixture of general MyST information and MyST-JS specific information, should be labelled or split.
* History and Background - I see that everything mystjs was renamed mystmd. Apparently the idea is that the Python/Sphinx implementation will just die and JS will reign supreme - this seems idealistic. It is pretty obvious that `myst-docutils-html` serves similar functions to the `mystmd` tool. I don't have any complaints about renaming the command-line tool, ideally `mystmd` would have the same interface whether it was the Python or JS implementation, but the actual projects should be clearly named and consistently distinguished from MyST as a Markdown format.
* Contributing - should include discussion of MyST-Py repos as well, link to https://myst-parser.readthedocs.io/en/latest/develop/index.html
* https://mystmd.org/spec - it should be more prominent, like at the top instead of the bottom. Maybe stuff in the "Reference" section could go here.
* https://myst-parser.readthedocs.io/en/latest/index.html - should more clearly distinguish between MyST and MyST-Py, also should link to main https://mystmd.org/ site
* https://myst-parser.readthedocs.io/en/latest/syntax/typography.html, siblings under 'Authoring' - should all be merged with https://mystmd.org/guide/ and redirected
* https://myst-parser.readthedocs.io/en/latest/configuration.html, siblings under 'Guide' - a mix of Sphinx-specific and general information, should be streamlined to Sphinx-specific information
* https://myst-parser.readthedocs.io/en/latest/develop/background.html - should be merged into https://mystmd.org/guide/background

**Comment by noirbizarre** [comment]
Hi 👋🏼 

Sorry to cross-post, but I found this post after posting [mine](https://github.com/orgs/jupyter-book/discussions/1712). Given I share the same confusion, I am just adding a comment there, so both threads are linked and discoverable.

**Comment by choldgraf** [comment]
Thanks for this question! I've provided an answer to a similar question at the link below (and will monitor the below link for follow-ups). I'll close this as a duplicate for now!

- https://github.com/orgs/jupyter-book/discussions/1712
